### PR TITLE
Add controller repository functions for getTokens

### PR DESCRIPTION
**Pull request by stringtheory1234**

### CODE REVIEW

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/controller/VitualQueueController.java [hunk 1]
```diff
@@ -36,4 +42,9 @@
     public CreateShopResponse addShop(@RequestBody CreateShopRequest createShopRequest) {
         return virtualQueueService.createShop(createShopRequest);
     }
+
+    @GetMapping("/token/{shopId}")
+    public List<Token> getTokens(@PathVariable UUID shopId) {
+		return virtualQueueService.getTokens(shopId);
+	}
```
Comment (KarishmaSinghChauhan) [inline backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/controller/VitualQueueController.java:49]: Similarly this closing bracket should be in line with the previous closing bracket (on line 44)

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/controller/VitualQueueController.java [hunk 2]
```diff
@@ -36,4 +42,9 @@
     public CreateShopResponse addShop(@RequestBody CreateShopRequest createShopRequest) {
         return virtualQueueService.createShop(createShopRequest);
     }
+
+    @GetMapping("/token/{shopId}")
+    public List<Token> getTokens(@PathVariable UUID shopId) {
+		return virtualQueueService.getTokens(shopId);
```
Comment (KarishmaSinghChauhan) [inline backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/controller/VitualQueueController.java:48]: I think this return statement should be in line (wrt indentation) with the previous return statement (on line 43). Please check once.

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/controller/VitualQueueController.java (hunk 1 above)
Comment (stringtheory1234) [reply backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/controller/VitualQueueController.java:49]: It's showing inline in my local code. Did u check by cloning it on your local machine?

Comment (KarishmaSinghChauhan) [reply backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/controller/VitualQueueController.java:48]: Yeah my bad. Cloned it and everything seems alright in my local machine.

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/controller/VitualQueueController.java (hunk 2 above)
Comment (stringtheory1234) [reply backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/controller/VitualQueueController.java:47]: this is showing on my local fine! please verify by cloning repo!

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetTokensForShopResponse.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+import com.google.gpay.virtualqueue.backendservice.model.Token;
+
+import lombok.Data;
+import lombok.Setter;
+
+@Data
+@Setter
+public class GetTokensForShopResponse {
+    private List<Token> tokenList = new ArrayList<>();
```
Comment (priytanvani) [inline backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetTokensForShopResponse.java:30]: Add field for shopId here?

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetTokensForShopResponse.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+import java.util.List;
+
+import com.google.gpay.virtualqueue.backendservice.model.Token;
+
+import lombok.Data;
+import lombok.Setter;
+
+@Data
+@Setter
+public class GetTokensForShopResponse {
```
Comment (priytanvani) [inline backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetTokensForShopResponse.java:29]: Do you think we could rename this as GetTokensResponse?

Comment (stringtheory1234) [reply backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetTokensForShopResponse.java]: Done!

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetTokensForShopResponse.java (hunk 3 above)
Comment (stringtheory1234) [reply backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetTokensForShopResponse.java]: Added!

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetTokensResponse.java [hunk 5]
```diff
@@ -26,6 +27,7 @@
 
 @Data
 @Setter
-public class GetTokensForShopResponse {
+public class GetTokensResponse {
     private List<Token> tokenList = new ArrayList<>();
+    private UUID shopId;
```
Comment (priytanvani) [inline backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetTokensResponse.java:32]: Nit. Let's move this up?

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetTokensResponse.java [hunk 6]
```diff
@@ -26,6 +27,7 @@
 
 @Data
 @Setter
-public class GetTokensForShopResponse {
+public class GetTokensResponse {
```
Comment (priytanvani) [inline backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetTokensResponse.java:30]: Let's have an @AllArgsConstructor and use that in the service?

Comment (stringtheory1234) [reply backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetTokensResponse.java:30]: Done!

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetTokensResponse.java (hunk 5 above)
Comment (stringtheory1234) [reply backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetTokensResponse.java:31]: Done!

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetTokensResponse.java [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+import com.google.gpay.virtualqueue.backendservice.model.Token;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class GetTokensResponse {
+    private UUID shopId;
+    private List<Token> tokenList = new ArrayList<>();
```
Comment (priytanvani) [inline backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetTokensResponse.java:32]: We should not have initializations in the proto classes.

Comment (stringtheory1234) [reply backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetTokensResponse.java]: Removed it!

**Pull request merged**